fsspec/implementations/tests/test_cached.py

```diff
@@ -37,6 +37,49 @@ def test_idempotent():
     assert fs3.storage == fs.storage
 
 
+@pytest.fixture()
+def disable_fs_caching():
```

Comment:
You probably just wanted `fs = FTPFileSystem(host, port, user, pw, skip_instance_cache=True)` instead of this fixture

Reply:
Sorry, I had not seen that option, I needed to use it only for the blockcache. If the tests pass, it should be ready to merge.